db/db_tunables.c

```diff
@@ -290,6 +290,8 @@ extern int gbl_osql_check_replicant_numops;
 extern int gbl_abort_on_missing_osql_session;
 extern int gbl_abort_irregular_set_durable_lsn;
 extern int gbl_legacy_schema;
+extern int gbl_selectv_writelock_on_update;
+extern int gbl_selectv_writelock;
```

Comment:
This one seems to be a duplicate so can be deleted.

Reply:
No - selectv_writelock acquires a writelock for every selectv, selectv_writelock_on_update only acquires a writelock for rows that are updated later